samples/QnAMaker/KnowledgebaseCrudSample.cs

```diff
@@ -0,0 +1,103 @@
+﻿using Microsoft.Azure.CognitiveServices.Knowledge.QnAMaker;
+using Microsoft.Azure.CognitiveServices.Knowledge.QnAMaker.Models;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.CognitiveServices.Samples.QnAMaker
+{
+    public class KnowledgebaseCrudSample
+    {
+        private IQnAMakerClient client;
+
+        public KnowledgebaseCrudSample(string apiKey, string endpoint)
+        {
+            this.client = new QnAMakerClient(new ApiKeyServiceClientCredentials(apiKey)) { Endpoint = endpoint };
+        }
+
+        public async Task Run()
```

Comment:
Please make this method static.

The whole static thing is to make the sample copy-paste-able. Customers in generally prefer to copy-paste and run instead of download our entire repo.

There should be two parameters for test methods: `key` and `endpoint`, (this is a limitation of our tester)